rules/windows/credential_access_mimikatz_powershell_module.toml

```diff
@@ -4,14 +4,14 @@ maturity = "development"
 updated_date = "2021/05/10"
 
 [rule]
-author = ["Elastic"]
+author = ["Elastic", "Austin Songer"]
```

Comment:
not to nitpick or take anything away from you, but in general, I think the author field corresponds to the contributors of the rule idea and logic.

Reply:
```suggestion
author = ["Elastic"]
```